fastify.js

```diff
@@ -350,7 +350,7 @@ function build (options) {
     if (cb === undefined) return listenPromise(port, address, backlog)
 
     fastify.ready(function (err) {
-      if (err) return cb(err)
+      if (err != null) return cb(err)
```

Comment:
Why `!=`?
`!== undefined || != null`?

Reply:
They are equivalent.

Reply:
from performance point of view too?

Reply:
Yes.

Reply:
Is it possible to check the performance of those tweaks in a reliable fashion?

Reply:
In this case is not a tweak. Is just "less characters for the same result" :P

Reply:
That was a serious question 😄 What tools are you using except autocannon?

Reply:
Oops! Sorry!
`console.time`, [`nanobench`](https://github.com/mafintosh/nanobench) and [`fastbench`](https://github.com/mcollina/fastbench) :D

Reply:
Thanks, 👍 same arsenal 😄

Reply:
@delvedor they are all time-based. I always looking for ops-per-second measurment but I think this is not provided by V8 right?